code/datums/components/material_container.dm

```diff
@@ -280,19 +280,20 @@
 		return FALSE
 
 	for(var/x in mats) //Loop through all required materials
-		var/datum/material/req_mat = x
-		if(!istype(req_mat))
-			if(ispath(req_mat)) //Is this an actual material, or is it a category?
-				req_mat = getmaterialref(req_mat) //Get the ref
+		if(!x == "Credits")
```

Comment:
```suggestion
		if(x != "Credits")
```

Reply:
That was there before the PR kinda rude why do I gotta fix it